components/main_menu/main_menu.jsx

```diff
@@ -343,6 +344,7 @@ class MainMenu extends React.PureComponent {
                             show={!this.props.mobile && this.props.enablePluginMarketplace}
                             dialogType={MarketplaceModal}
                             text={formatMessage({id: 'navbar_dropdown.marketplace', defaultMessage: 'Plugin Marketplace'})}
+                            showUnread={!this.props.firstAdminVisitMarketplaceStatus}
```

Comment:
Should we move all the logic here and also check if the user is an admin?

Reply:
Do we have to though? This option only shows if the user is an admin, no?